docs/test-cases/vcf-highlight/Conditions/Highlight vcf with simple conditions.md

```diff
@@ -0,0 +1,78 @@
+# Highlighting variations depending on the set conditions
```

Comment:
`Highlight variations depending on the set of conditions` or `Highlight variations depending on the selected profile`